.github/workflows/update-reproducibles.yml

```diff
@@ -0,0 +1,42 @@
+name: Update reproducible requirements
+on:
+  pull_request:
+    branches: [develop]
+    paths:
+      - requirements/base.txt # run this GHA only if requirements file is changed
```

Comment:
just to make sure, we do not need this GHA to be executed in case of the dependencies got updated, or their dependencies got updated. Dependabot should handle that, right?

Reply:
I wanted to keep our existing flow where we update these reproducibles when we update our requirements (for example when we bump SAMT version). If we run this for every PR there is a chance that we will have updates every day or so (for minor version updates), which might make harder for those PRs getting merged, and I will leave those updates to dependabot as it is been currently handled.